### DISCUSSION TITLE
Why does it say that "You have been idle for <time>" even if I'm working?

### DISCUSSION THREAD

**Discussion by Prayag2** [super-productivity/super-productivity#2086] [Q&A] [answered]
So, I really like this app but there's this one thing that I can't understand. Even if I'm working, it says that I've been idle for some time. Why is it happening? Even if I'm tracking time using the Pomodoro timer, a popup appears saying "You have been idle for <time>" and then I have to click on "Task" to get rid of it. Why?
Thanks in advance!

**Comment by johannesjo** [comment]
Hey there! This looks like a bug. Are you using Wayland by any chance? There are some issues on Wayland with the idle detection unfortunately.

**Reply by Prayag2** [reply]
I don't think this is a bug. I think this is happening because I'm not actually working on my device but on my notebook (I mean real notebooks not laptops). I'm using X11.

**Reply by johannesjo** [reply]
Ahh! I think I understand. This is kind of intentional and hard to solve right for all cases. Super Productivity assumes that you're idle, if you're away from the computer. If you dislike this, you can deactivate idle handling completely.

**Reply by Prayag2** [reply]
Okay, thanks!

**Comment by Prayag2** [comment, marked as answer]
Answer: It was not a bug. It was happening because I was working on paper instead of the PC so I'm closing this now :)
